src/main/java/repo/impl/PostgresTokenRepo.java

```diff
@@ -0,0 +1,74 @@
+package repo.impl;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+import repo.TokenRepo;
+import util.Constants;
+
+import java.io.*;
+import java.sql.*;
+import java.sql.Date;
+import java.util.*;
+
+/**
+ * DAO implementation for HQ's django_session key table.
+ */
+@Repository
+public class PostgresTokenRepo implements TokenRepo{
+
+    private final Log log = LogFactory.getLog(PostgresTokenRepo.class);
+
+    @Autowired
+    @Qualifier("jdbcHq")
+    private JdbcTemplate jdbcTemplate;
+
+    @Override
+    public boolean isAuthorized(String tokenId) {
+        String sql = String.format("SELECT * FROM %s WHERE session_key = ?",
```

Comment:
nit, but you could just include the the date check in the where clause instead of checking it after